test/features/standup.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Standup-module
+
+    Scenario: get list of standups
+        Given make a GET request to "/standup"
+        Then the response status code should be 200
+        And the response should be "[]"
+
+    Scenario: add a new standup
+        Given make a POST request to "/standup" with:
+            | discordUser      | "eddiehubber"          |
+            | yesterdayMessage | "yesterday I did this" |
+            | todayMessage     | "Today I'll do this"   |
+        Then the response status code should be 201
+        And the response should contains:
+            | discordUser      | "eddiehubber"              |
+            | yesterdayMessage | "yesterday I did this"     |
+            | todayMessage     | "Today I'll do this"       |
+            | createdOn        | "2021-01-01T00:00:00.000Z" |
+            | id               | 666                        |
+
+
+    //TODO Step Definition Array of Objects
```

Comment:
Thats the Test for the Search Endpoint, which returns an Array ob Standup-Objects.